test/Microsoft.Dnx.Tooling.FunctionalTests/DnuPackTests.cs

```diff
@@ -73,6 +80,8 @@ public void P2PDifferentFrameworks(DnxSdk sdk)
 
             // Assert
             Assert.Equal(0, result.ExitCode);
+
+            TestUtils.CleanUpTestDir<DnuPackTests>(sdk);
```

Comment:
This makes sense, I'll make a point for it in #2825 . We should revisit this implementation of this feature in the future but not in this PR.